src/module-vsbridge-indexer-agreement/Index/Mapping/Agreement.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Copyright Divante Sp. z o.o.
+ * See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Divante\VsbridgeIndexerAgreement\Index\Mapping;
+
+use Divante\VsbridgeIndexerCore\Api\Mapping\FieldInterface;
+use Divante\VsbridgeIndexerCore\Api\MappingInterface;
+
+class Agreement implements MappingInterface
+{
+    /**
+     * @return array|mixed|null
+     */
+    public function getMappingProperties()
+    {
+        $properties = [
+            'id' => ['type' => FieldInterface::TYPE_LONG],
+            'content' => ['type' => FieldInterface::TYPE_TEXT],
+            'active' => ['type' => FieldInterface::TYPE_BOOLEAN],
+            'name' => ['type' => FieldInterface::TYPE_TEXT],
```

Comment:
what do you think of switching 'name' to 'title'?

Reply:
My way of thinking is that it should be the same name as in the magento database, but if you feel that 'title' is better in this case I can change it. Please confirm that I should change it

Reply:
just like we talked on priv, I changed to 'title'